docs/best_practices.md

```diff
@@ -178,10 +177,12 @@ We use [Jest](https://jestjs.io/) for our unit tests, and [Mocha](https://mochaj
 
 Some top-level notes:
 
-- ~~We use [`enzyme`](https://enzymejs.github.io/enzyme/)~~
-- As of 2021, we've begun using [@testing-library/react](https://testing-library.com/docs/) for our tests. (See [this doc](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library) for some common mistakes and best practices.)
+- We use [`enzyme`](https://enzymejs.github.io/enzyme/)
+- As of 2021, we've also begun using [@testing-library/react](https://testing-library.com/docs/) for our tests. (See [this doc](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library) for some common mistakes and best practices.)
+- Enzyme is a unit testing library that matches React's internals 1:1. `@testing-library/react` is fundamentally a light weight html-based integration testing library. Depending on what you want to test one might be better suited than the other. Use what you think works best!
+- When testing React Hooks, [`@testing-library/react-hooks`](https://www.npmjs.com/package/@testing-library/react-hooks) is the best way to test that level of the stack.
 - We avoid snapshot tests; they produce too much churn for too little value.
-- We use the `relay-test-utils` package for testing Relay code, and [this helper](https://github.com/artsy/force/blob/0b291f005763e7c2600a5077786c9510bf655079/src/v2/DevTools/setupTestWrapper.tsx) for quickly spinning up tests. Note that this helper can't test `QueryRenderer`s; extract the render code into a fragment-like container and test that. (See the [`RegisterButton` component](https://github.com/artsy/force/blob/main/src/v2/Apps/Auction2/Components/AuctionDetails/__tests__/RegisterButton.jest.tsx#L1) for an example.)
+- We use the `relay-test-utils` package for testing Relay code, and [this helper](https://github.com/artsy/force/blob/0b291f005763e7c2600a5077786c9510bf655079/src/v2/DevTools/setupTestWrapper.tsx) for quickly spinning up tests. There are two versions, one for Enzyme, and one for RTL. Note that this helper can't test `QueryRenderer`s; extract the render code into a fragment-like container and test that. (See the [`RegisterButton` component](https://github.com/artsy/force/blob/main/src/v2/Apps/Auction2/Components/AuctionDetails/__tests__/RegisterButton.jest.tsx#L1) for an example.)
```

Comment:
I wonder if it's worth of mentioning the buildPage abstraction we have there for some heavy tests

Reply:
Nah, that pattern should be avoided I think (not that it ever will, as so many tests rely on it!) just because it's built on some highly complicated custom relay testing infra that we wrote from scratch before relay-test-tools was released. Ideally all of the order app tests can be migrated to more official implementations at some point.